todoman/ui.py

```diff
@@ -303,6 +305,19 @@ def format_datetime(self, dt):
             self._format_time(time_part)
         )))
 
+    def parse_priority(self, priority):
+        if priority not in ['low', 'high', 'medium', 'none']:
```

Comment:
To be clear, something like:

    if priority == 'low':
        ...
    elif priority == 'medium':
        ...
    ...
    else:
        raise ValueError('Priority has to be one of ...')